core/server/worker/src/main/java/alluxio/worker/page/PagedBlockStore.java

```diff
@@ -128,7 +133,11 @@ public void commitBlock(long sessionId, long blockId, boolean pinOnCreate)
   @Override
   public String createBlock(long sessionId, long blockId, int tier,
       CreateBlockOptions createBlockOptions) throws WorkerOutOfSpaceException, IOException {
-    return null;
+    //TODO(Beinan): port the allocator algorithm from tiered block store
+    PageStoreDir pageStoreDir = mPageStoreDirs.get(
+        Math.floorMod(Long.hashCode(blockId), mPageStoreDirs.size()));
+    pageStoreDir.putTempFile(String.valueOf(blockId));
+    return "DUMMY_FILE_PATH";
```

Comment:
What should be returned here if the page store is an in-memory store or a rocksdb store?

Reply:
Great question, I've no idea so far.  But actually the caller does not need this path except the short circuit read.